with-legend-state-supabase/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "expo-supa-legend-state",
+  "version": "1.0.0",
+  "main": "expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
+    "@expo/metro-runtime": "~3.2.1",
+    "@legendapp/state": "^3.0.0-alpha.37",
+    "@supabase/supabase-js": "^2.45.1",
+    "@types/react": "~18.2.79",
+    "@types/uuid": "^10.0.0",
+    "expo": "^51.0.24",
+    "i": "^0.3.7",
+    "npm": "^10.8.2",
```

Comment:
Is it necessary to install `npm` as a dependency here?

Reply:
Not at all, thanks for catching these. not sure how they got there 😬